looker_deployer/commands/deploy_content.py

```diff
@@ -19,28 +19,36 @@
 import tempfile
 import shutil
 import threading
+import json
 from concurrent.futures import ThreadPoolExecutor
 from itertools import repeat
 from looker_deployer.utils import deploy_logging
 from looker_deployer.utils import parse_ini
 from looker_deployer.utils.get_client import get_client
-from looker_sdk import models
+from looker_sdk import models40 as models
 
 
 logger = deploy_logging.get_logger(__name__)
 
+def alert_cleanup(sdk):
+    logger.debug("cleaning up orphaned alerts")
+    disabled_alerts = sdk.search_alerts(disabled="true")
+    for alert in disabled_alerts:
+        if alert['disabled_reason'] == "Dashboard element has been removed.":
+            sdk.delete_alert(alert['id'])
```

Comment:
Also logger.info for any time we do delete or updating so people can see which IDs changed